### PR TITLE
Added Songbook endpoint

### PR DESCRIPTION
**Pull request by jackconwayiv**
Added the songbooks endpoint and pluralized the song_entries endpoint at Sam's request.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -137,4 +137,7 @@ dmypy.json
 .pytype/
 
 # Cython debug symbols
-cython_debug/
\ No newline at end of file
+cython_debug/
+
+# VScode Settings
+.vscode/
```
Comment (samdatkins) [inline .gitignore:143]: I had Jack add this so git doesn't pick up our local vs code settings

File: api/urls.py [hunk 2]
```diff
@@ -3,10 +3,12 @@
 
 from api.views.song import SongViewSet
 from api.views.song_entry import SongEntryViewSet
+from api.views.songbook import SongbookViewSet
 
 router = routers.DefaultRouter()
 
 router.register(r"songs", SongViewSet)
-router.register(r"song_entry", SongEntryViewSet)
+router.register(r"song_entries", SongEntryViewSet)
```
Comment (samdatkins) [inline api/urls.py:11]: And I asked him to pluralize this for consistency (and generally, plurals are the accepted way to name endpoints)

**Comment by samdatkins** [comment]
@jackconwayiv feel free to merge at your leisure

**Comment by jackconwayiv** [comment]
Roger, was just waiting for Drew.

It there!

**Pull request merged**